### PR TITLE
Allow to disable new chat features through the param `disabledFeatures`

### PR DESCRIPTION
**Pull request by gustavotrott**
Allow to disable the new chat features through the param `disabledFeatures`.

- `deleteChatMessage`
- `editChatMessage`
- `replyChatMessage`
- `chatMessageReactions`

It affects only the backend once the frontend portion is under development.

### CODE REVIEW

File: bigbluebutton-web/grails-app/conf/bigbluebutton.properties [hunk 1]
```diff
@@ -453,7 +453,8 @@ endWhenNoModeratorDelayInMinutes=1
 # List of features to disable (comma-separated)
 # https://docs.bigbluebutton.org/3.0/development/api/#create
 # Available options:
-# chat, sharedNotes, polls, screenshare, externalVideos, layouts, captions, liveTranscription,
+# sharedNotes, polls, screenshare, externalVideos, layouts, captions, liveTranscription,
+# chat, deleteChatMessage, editChatMessage, replyChatMessage, chatMessageReactions
```
Comment (antobinary) [inline bigbluebutton-web/grails-app/conf/bigbluebutton.properties:457]: Please sort alphabetically, possibly each item on new line :pray:

File: docs/docs/data/create.tsx [hunk 2]
```diff
@@ -350,7 +350,135 @@ const createEndpointTableData = [
...
+    "description": (
+        <>
+          List (comma-separated) of features to disable in a particular meeting. (added 2.5)
+          <br /><br />
+          Available options to disable:
```
Comment (antobinary) [inline docs/docs/data/create.tsx:357]: Very much in favor of this suggestion. Thanks, Bunny!

File: docs/docs/data/create.tsx [hunk 3]
```diff
@@ -350,7 +350,135 @@ const createEndpointTableData = [
...
+          <ul>
+            <li>
+              <code className="language-plaintext highlighter-rouge">
+                breakoutRooms
+              </code> - <b>Breakout Rooms</b>
```
Comment (antobinary) [inline docs/docs/data/create.tsx:362]: Please also document in https://github.com/bigbluebutton/bigbluebutton/blob/v3.0.x-release/docs/docs/development/api.md?plain=1#L114

**Comment by antobinary** [comment]
Actually, I think it will be best if we don't yet document in
api.md
create.tsx
bigbluebutton.properties

We should wait until we can support these in recording #21356 , playback, (ideally testing too)

**Comment by gustavotrott** [comment]
@JoVictorNunes please make the frontend respect these new `disabledFeatures` options! I implemented only the backend portion.

**Review by antobinary — APPROVED** [review]
LGTM

**Pull request merged**